CHANGELOG.md

```diff
@@ -1,3 +1,49 @@
+# 94.1.0
+
+This minor release intoduces new API to FlexibleHeader, SnackBar and Ripple. It also includes bug fixes for multiple components.
```

Comment:
Do we want to call-out the appearance change for MDCSnackbar from legacy to 2018 as a default?

Reply:
Done